cooperator_worker/models/res_partner.py

```diff
@@ -1,28 +1,30 @@
-# Copyright 2019-2020 Coop IT Easy SC
+# Copyright 2022 Coop IT Easy SCRLfs
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import _, api, fields, models
-from odoo.exceptions import ValidationError
+from odoo import api, fields, models
 
 
 class Partner(models.Model):
     _inherit = "res.partner"
 
     # this field is not displayed in the view
     # cf issue https://github.com/beescoop/Obeesdoo/issues/374
+    # force is_worker to True
     worker_store = fields.Boolean(
         string="Force Worker",
         help="Check to subscribe member to their shift even"
         " if they are not yet effective cooperator.",
     )
-    info_session_confirmed = fields.Boolean(
-        string="Confirmed presence to info session", default=False
-    )
+
     is_worker = fields.Boolean(
         compute="_compute_is_worker",
         search="_search_worker",
         readonly=True,
+        # override parent definition
         related="",
+        help="This field can only be set if Create Subscription "
+        "button was clicked on this user and payment was "
+        "validated, for a Share Type where Owner-can-work is checked ",
```

Comment:
I have difficulty understanding the help message. Is it possible to make it clearer?